### PR TITLE
Add device templates for Cortex-A53 and Cortex-R4, R5, R7, R8

### PR DESCRIPTION
**Pull request by Masmiseim36**

### CODE REVIEW

**Review by JonatanAntoni — APPROVED** [review]

File: Device/ARMCA35/Config/mem_ARMCA35.h [hunk 1]
```diff
@@ -0,0 +1,100 @@
+/**************************************************************************//**
+ * @file     mem_ARMCA35.h
+ * @brief    Memory base and size definitions (used in scatter file)
+ * @version  V1.0.0
+ * @date     26. Mai 2024
```
Comment (ghost) [inline Device/ARMCA35/Config/mem_ARMCA35.h:5]: it is a little petty but please write 'May' instead 'Mai'.

File: Device/ARMCA35/Include/ARMCA35.h [hunk 2]
```diff
@@ -0,0 +1,184 @@
+/**************************************************************************//**
+ * @file     ARMCA35.h
+ * @brief    CMSIS-Core(A) Device Header File for Device ARMCA35
+ *
+ * @version  V1.0.0
+ * @date     26. Mai 2024
```
Comment (ghost) [inline Device/ARMCA35/Include/ARMCA35.h:6]: please write 'May' instead 'Mai'.

File: Device/ARMCA35/Include/system_ARMCA35.h [hunk 3]
```diff
@@ -0,0 +1,65 @@
+/******************************************************************************
+ * @file     system_ARMCA35.h
+ * @brief    CMSIS Device System Header File for Arm Cortex-A9 Device Series
+ * @version  V1.0.0
+ * @date     26. Mai 2024
```
Comment (ghost) [inline Device/ARMCA35/Include/system_ARMCA35.h:5]: please write 'May' instead 'Mai'.

File: Device/ARMCA35/Source/startup_ARMCA35.c [hunk 4]
```diff
@@ -0,0 +1,59 @@
+/******************************************************************************
+ * @file     startup_ARMCA35.c
+ * @brief    CMSIS Device System Source File for Arm Cortex-A9 Device Series
+ * @version  V1.0.0
+ * @date     26. Mai 2024
```
Comment (ghost) [inline Device/ARMCA35/Source/startup_ARMCA35.c:5]: please write 'May' instead 'Mai'.

File: Device/ARMCA35/Source/system_ARMCA35.c [hunk 5]
```diff
@@ -0,0 +1,93 @@
+/******************************************************************************
+ * @file     system_ARMCA35.c
+ * @brief    CMSIS Device System Source File for Arm Cortex-A9 Device Series
+ * @version  V1.0.0
+ * @date     26. Mau 2024
```
Comment (ghost) [inline Device/ARMCA35/Source/system_ARMCA35.c:5]: please write 'May' instead 'Mau'.

File: Device/ARMCA55/Config/mem_ARMCA55.h [hunk 6]
```diff
@@ -0,0 +1,100 @@
+/**************************************************************************//**
+ * @file     mem_ARMCA55.h
+ * @brief    Memory base and size definitions (used in scatter file)
+ * @version  V1.0.0
+ * @date     26. Mai 2024
```
Comment (ghost) [inline Device/ARMCA55/Config/mem_ARMCA55.h:5]: please write 'May' instead 'Mai'.

File: Device/ARMCA55/Include/ARMCA55.h [hunk 7]
```diff
@@ -0,0 +1,184 @@
+/**************************************************************************//**
+ * @file     ARMCA55.h
+ * @brief    CMSIS-Core(A) Device Header File for Device ARMCA55
+ *
+ * @version  V1.0.0
+ * @date     26. Mai 2024
```
Comment (ghost) [inline Device/ARMCA55/Include/ARMCA55.h:6]: please write 'May' instead 'Mai'.

File: Device/ARMCA55/Include/system_ARMCA55.h [hunk 8]
```diff
@@ -0,0 +1,65 @@
+/******************************************************************************
+ * @file     system_ARMCA55.h
+ * @brief    CMSIS Device System Header File for Arm Cortex-A9 Device Series
+ * @version  V1.0.0
+ * @date     26. Mai 2024
```
Comment (ghost) [inline Device/ARMCA55/Include/system_ARMCA55.h:5]: please write 'May' instead 'Mai'.

File: Device/ARMCA55/Source/startup_ARMCA55.c [hunk 9]
```diff
@@ -0,0 +1,59 @@
+/******************************************************************************
+ * @file     startup_ARMCA55.c
+ * @brief    CMSIS Device System Source File for Arm Cortex-A9 Device Series
+ * @version  V1.0.0
+ * @date     26. Mai 2024
```
Comment (ghost) [inline Device/ARMCA55/Source/startup_ARMCA55.c:5]: please write 'May' instead 'Mai'.

File: Device/ARMCA55/Source/system_ARMCA55.c [hunk 10]
```diff
@@ -0,0 +1,93 @@
+/******************************************************************************
+ * @file     system_ARMCA55.c
+ * @brief    CMSIS Device System Source File for Arm Cortex-A9 Device Series
+ * @version  V1.0.0
+ * @date     26. Mau 2024
```
Comment (ghost) [inline Device/ARMCA55/Source/system_ARMCA55.c:5]: please write 'May' instead 'Mau'.

File: Device/ARMCA57/Config/mem_ARMCA57.h [hunk 11]
```diff
@@ -0,0 +1,100 @@
+/**************************************************************************//**
+ * @file     mem_ARMCA57.h
+ * @brief    Memory base and size definitions (used in scatter file)
+ * @version  V1.0.0
+ * @date     26. Mai 2024
```
Comment (ghost) [inline Device/ARMCA57/Config/mem_ARMCA57.h:5]: please write 'May' instead 'Mai'.

File: Device/ARMCA57/Include/ARMCA57.h [hunk 12]
```diff
@@ -0,0 +1,184 @@
+/**************************************************************************//**
+ * @file     ARMCA57.h
+ * @brief    CMSIS-Core(A) Device Header File for Device ARMCA57
+ *
+ * @version  V1.0.0
+ * @date     26. Mai 2024
```
Comment (ghost) [inline Device/ARMCA57/Include/ARMCA57.h:6]: please write 'May' instead 'Mai'.

File: Device/ARMCA57/Include/system_ARMCA57.h [hunk 13]
```diff
@@ -0,0 +1,65 @@
+/******************************************************************************
+ * @file     system_ARMCA57.h
+ * @brief    CMSIS Device System Header File for Arm Cortex-A9 Device Series
+ * @version  V1.0.0
+ * @date     26. Mai 2024
```
Comment (ghost) [inline Device/ARMCA57/Include/system_ARMCA57.h:5]: please write 'May' instead 'Mai'.

File: Device/ARMCA57/Source/startup_ARMCA57.c [hunk 14]
```diff
@@ -0,0 +1,59 @@
+/******************************************************************************
+ * @file     startup_ARMCA57.c
+ * @brief    CMSIS Device System Source File for Arm Cortex-A9 Device Series
+ * @version  V1.0.0
+ * @date     26. Mai 2024
```
Comment (ghost) [inline Device/ARMCA57/Source/startup_ARMCA57.c:5]: please write 'May' instead 'Mai'.

File: Device/ARMCA57/Source/system_ARMCA57.c [hunk 15]
```diff
@@ -0,0 +1,93 @@
+/******************************************************************************
+ * @file     system_ARMCA57.c
+ * @brief    CMSIS Device System Source File for Arm Cortex-A9 Device Series
+ * @version  V1.0.0
+ * @date     26. Mau 2024
```
Comment (ghost) [inline Device/ARMCA57/Source/system_ARMCA57.c:5]: please write 'May' instead 'Mau'.

**Review by ghost — COMMENTED** [review]
Please correct the month name used for @date to 'May'

**Pull request merged**